tests/images/debian-systemd/files/bootstrap.sh

```diff
@@ -135,6 +135,7 @@ install_via_apt() {
         c8y-configuration-plugin \
         c8y-log-plugin \
         tedge-watchdog
+    apt-get install -y c8y-firmware-plugin
```

Comment:
I think we need to raise a ticket to address this dependency issue that we have with the `c8y-firmware-plugin`.

Reply:
Created: https://github.com/thin-edge/thin-edge.io/issues/1793